site/features.md

```diff
@@ -6,44 +6,70 @@ menu_order: 15
+
+##<a name="mode"></a>Graphic or Table Mode
+
+In addition to these views, nodes can be presented either in graphical or in table mode. The graphical mode is practical for obtaining a quick visual overview of your app, its infrastructure and connections between all of the nodes. And when you switch to table mode, nodes are presented in a convenient list that displays the resources being consumed by processes, containers, and hosts by dynamically shifting the resource heavy nodes to the top of the table, much like the UNIX `top` command does. 
+
+!['Table Mode Displaying Nodes by resource usage'](images/table-mode.png)
 
-!['Automatic Topology Mapping'](images/topology.png)
+##<a name="flexible-filtering"></a>Flexible Filtering
 
-##<a name="flexible-view-filtering"></a>Flexible Filtering
+In the left-hand corner of the UI are other filtering and other options. Nodes can be filtered by CPU and Memory so that you can easily find containers using the most resources. In the container view, options are available to filter by system, application or to show all of the containers and if you are running an app in Kubernetes then your app can be filtered by namespace and by container state whether running or stopped or contained and uncontained. 
 
-Nodes can be filtered by various properties, for example in the Container view, you can filter by System Containers vs Application Containers or by Stopped vs. Running Containers. Scope can also display various metrics such as CPU and Memory usage in the nodes, for example allowing you to easily find the container using the most CPU or memory.  Mouse-over any of the nodes to view its filtered metric at a glance.
+!['CPU usage indicator in Weave Scope'](images/hosts-cpu.png)
 
-###<a name="powerful-search"></a> Powerful Search
+##<a name="powerful-search"></a> Powerful Search
 
-Powerful search capability enables you to find node types, containers and processes by name, label or even path.  The search functionality supports simple operands so that you can for example, find processes consuming a certain Memory limit or nodes using too much CPU. You can also stack filters to create custom, complex search criterion. 
+Powerful search capability enables you to find node types, containers and processes by name, label or even path.  The search functionality supports simple operands so that you can for example, find processes consuming a certain Memory limit or nodes using too much CPU. Search terms may also be stacked to create custom, complex search criterion. See the online help for a full list of allowable operands.
```

Comment:
By _online help_ you mean in the app itself?

* If yes, please try to make it more clear
* If not, please include a link to the documemtation you are referring to.